user_guide_src/source/libraries/uploaded_files.rst

```diff
@@ -83,9 +83,9 @@ this code and save it to your **app/Controllers** directory:
 
 .. literalinclude:: uploaded_files/002.php
 
-Since the value of a file upload HTML field doesn't exist, and is stored in the
+Since HTML file element has no value attribute, and the upload file is stored in the
```

Comment:
The original sentence is also difficult to understand, but this also seems inaccurate.
The original says there is no data for uploaded files in `$_POST`.

I don't understand what "HTML file element has no value attribute" says.

Reply:
I sent another PR to improve the description. See #8910